src/components/Navbar.vue

```diff
@@ -1,58 +1,44 @@
 <script setup lang="ts">
-    import { ref } from 'vue';
-    // Définition du type pour les Boutons et commandes
-    interface Bouton {
-    titre: string;
-    link: string;
-    perm: string;
-    }
+import { ref } from 'vue';
 
-    // Utilisation de 'ref' pour la réactivité. Vous pouvez aussi utiliser 'reactive' pour des objets complexes.
-    const boutons = ref<Bouton[]>([
-        { titre: "Acceuil", link: "/" , perm: "all" },
-        { titre: "Recherche", link: "/recherche" , perm: "all" },
-        { titre: "Playlist", link: "/playlist" , perm: "all" },
-        { titre: "Abonnements", link: "/abonnements" , perm: "all" },
-        { titre: "Dashboard", link: "/dashboard" , perm: "admin" },
-        { titre: "Gestion comptes", link: "/gestion-comptes" , perm: "admin" }
-    ]);
+interface Bouton {
+titre: string;
+link: string;
+perm: string;
+}
 
- 
+const boutons = ref<Bouton[]>([
+    { titre: "Acceuil", link: "/" , perm: "all" },
+    { titre: "Recherche", link: "/recherche" , perm: "all" },
+    { titre: "Playlist", link: "/playlist" , perm: "all" },
+    { titre: "Abonnements", link: "/abonnements" , perm: "all" },
+    { titre: "Dashboard", link: "/dashboard" , perm: "admin" },
+    { titre: "Gestion comptes", link: "/gestion-comptes" , perm: "admin" }
+]);
 
-    const commandes = ref<Bouton[]>([]);
-    const searchShowed = ref<boolean>(false);
-    const menunotif = ref<boolean>(false);
-    const menucompte = ref<boolean>(false);
-    const logoPath = ref(' /assets/logo.png');
+const searchShowed = ref<boolean>(false);
+const menunotif = ref<boolean>(false);
+const menucompte = ref<boolean>(false);
+const logoPath = ref(' /logo.png');
```

Comment:
Remove trailing space
```suggestion
const logoPath = ref('/logo.png');
```